server/models/User.js

```diff
@@ -49,6 +49,9 @@ const userSchema = new Schema({
         price_level: {
           type: Number,
         },
+        photo_reference: {
+          type: String,
+        },
```

Comment:
You can also add the other data points from the FavRestaurantData interface here if you want: lat: number, lng: number, rating: number, types: [string].

Reply:
I agree. Changes made, please let me know what you think.